react/features/base/dialog/components/Dialog.web.js

```diff
@@ -0,0 +1,267 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import AKButton from '@atlaskit/button';
+import AKButtonGroup from '@atlaskit/button-group';
+import ModalDialog from '@atlaskit/modal-dialog';
+
+import { hideDialog } from '../actions';
+import { translate } from '../../i18n';
+
+/**
+ * Web dialog that uses atlaskit modal-dialog to display dialogs.
+ */
+class Dialog extends Component {
```

Comment:
There seems to be some duplication here, how about creating an AbstractDialog and putting the common bits there?